### PR TITLE
First add for Prob_1

### PR DESCRIPTION
**Pull request by MMMany**
1) Upload datasets
2) Py code for Prob_1 (target dataset : HTS 2020)

### CODE REVIEW

**Comment by MMMany** [comment]
[Pull Request : First add for Prob_1 #1](https://trello.com/c/eFmn8WfH/33-pull-request-first-add-for-prob1-1)

**Comment by suhyeon-joo** [comment]
well done

**Comment by suhyeon-joo** [comment]
Review

File: Prob-1_Barrel.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.preprocessing import LabelEncoder
+import time, os
+from multiprocessing import Manager
+from concurrent.futures import ProcessPoolExecutor as PPE
+
+# MLB Barrel 기준(AVG 0.5, SLG 1.5) 만족하는 '타구속도, 발사각도 범위, 타율, 장타율' 계산 함수
+# 병렬처리 함수
+def work(df, v, values, partition, columns, result):
```
Comment (MMMany) [inline Prob-1_Barrel.py:11]: LGTM

**Review by souvenir19 — APPROVED** [review]
well done

**Review by suhyeon-joo — APPROVED** [review]
I review you

**Review by souvenir19 — COMMENTED** [review]
confirmed.

**Pull request merged**